commit cdfb5006adb2f1b8184443bbae1b1b60190927f4
Author: haolong.zhang <haolong.zhang@ck-telecom.com>
Date:   Sat Aug 16 11:35:34 2014 +0800

    [Feature] Enable WIFI/BT/GPS factory test fucntion
    
        [id]:N/A
        [TYPE]
           FEATURE
        [Products]
           hicam_baseline
        [Requirement]
           N/A
        [Solution Description]
           N/A
        [Other info]
    	修改:         mediatek/factory/src/factory.c
    
    Change-Id: I13ec45bf343c928eba061817ba85e28c0faa3f1e

diff --git a/mediatek/factory/src/factory.c b/mediatek/factory/src/factory.c
index 7da4341..858eb22 100644
--- a/mediatek/factory/src/factory.c
+++ b/mediatek/factory/src/factory.c
@@ -416,7 +416,7 @@ static item_t ftm_test_items[] = {
 #endif
 #ifdef MTK_GPS_SUPPORT
 #ifdef FEATURE_FTM_GPS
-    //item(ITEM_GPS,     uistr_gps),
+    item(ITEM_GPS,     uistr_gps),
 #endif
 #endif
 
@@ -439,13 +439,13 @@ static item_t ftm_test_items[] = {
 
 #ifdef MTK_BT_SUPPORT
 #ifdef FEATURE_FTM_BT
-   // item(ITEM_BT, uistr_bluetooth),
+    item(ITEM_BT, uistr_bluetooth),
 #endif
 #endif
 
 #ifdef MTK_WLAN_SUPPORT
 #ifdef FEATURE_FTM_WIFI
-    //item(ITEM_WIFI, uistr_wifi),
+    item(ITEM_WIFI, uistr_wifi),
 #endif
 #endif
 
